From 4113668997af81e1fc10bb3d1bbe0f68d19d7499 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Mon, 10 Feb 2025 11:04:51 +0800
Subject: [PATCH 2/2] AOSCOS: fix: remove rb-fsevent dependency

This is a macOS-specific dependency for hookup to their FSEvent API.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 sass-listen.gemspec | 1 -
 1 file changed, 1 deletion(-)

diff --git a/sass-listen.gemspec b/sass-listen.gemspec
index 82cce42..80b1cf1 100644
--- a/sass-listen.gemspec
+++ b/sass-listen.gemspec
@@ -22,7 +22,6 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 1.9.3'
 
-  s.add_dependency 'rb-fsevent', '>= 0.9', '>= 0.9.4'
   s.add_dependency 'rb-inotify', '>= 0.9', '>= 0.9.7'
 
   s.add_development_dependency 'bundler', '>= 1.3.5'
-- 
2.48.1

